src/ui/views/TradingBlock.tsx

```diff
@@ -173,19 +177,65 @@ const pickCols = getCols("", "Draft Picks");
 pickCols[0].sortSequence = [];
 pickCols[1].width = "100%";
 
+const populateExtendedPositions = (items: string[]): string[] => {
+	let extPos = [...items].filter(pos => !["GF", "FC", "F", "G"].includes(pos));
+	if (extPos.includes("PG") || extPos.includes("SG")) extPos.push("G");
+	if (extPos.includes("PF") || extPos.includes("SF")) extPos.push("F");
+	if (extPos.includes("PF") || extPos.includes("C")) extPos.push("FC");
+	if (extPos.includes("SF") || extPos.includes("SG")) extPos.push("GF");
```

Comment:
Wrap in `if (isSport("basketball")) { ... }` so this code doesn't get included in other sports. (I know, tiny file size difference, but every bit counts!)

Reply:
haha for sure, the size of the game isn't going to get smaller over time, so make sense